aiida/engine/processes/calcjobs/manager.py

```diff
@@ -189,63 +211,72 @@ def _has_job_state_changed(old, new):
         return old.job_state != new.job_state or old.job_substate != new.job_substate
 
     def _get_next_update_delay(self):
-        """
-        Calculate when we are next allowed to call the scheduler get jobs command
-        based on when we last called it, how long has elapsed and the minimum given
-        update interval.
+        """Calculate when we are next allowed to call the scheduler get jobs command.
 
-        :return: The delay (in seconds) for when it's safe to call the get jobs command
+        This delay will be calculated as the minimum safe interval that needs to be guaranteed, as defined by the
+        authentication info for this instance, minus the last time we updated the list.
+
+        :return: delay (in seconds) after which the minimum update interval has passed since last update
         :rtype: float
         """
-        if self._last_updated is None:
+        if self.last_updated is None:
+            self.logger.info('AuthInfo<{}>: next scheduler update will be in {} seconds'.format(self._authinfo.pk, 0))
             # Never updated, so do it straight away
             return 0.
 
-        # Make sure to actually 'get' it here, so that if the user changed it
-        # between times we use the current value
+        # Make sure to actually 'get' it here, so that if the user changed it between times we use the current value
         minimum_interval = self._authinfo.computer.get_minimum_job_poll_interval()
-        elapsed = time.time() - self._last_updated
+        elapsed = time.time() - self.last_updated
+
+        delay = max(minimum_interval - elapsed, 0.)
+        self.logger.info('AuthInfo<{}>: next scheduler update will be in {} seconds'.format(self._authinfo.pk, delay))
```

Comment:
```suggestion
        self.logger.info('AuthInfo<{}>: next scheduler update allowed in {} seconds'.format(self._authinfo.pk, delay))
```

Reply:
Even though your change is correct here, it made me realize that it is better to put a single info message at the point when the actual update is performed.